src/documentation/constants/utilities.ts

```diff
@@ -0,0 +1,13 @@
+const redirectionScript = (chiVersion: string): string => {
```

Comment:
```suggestion
import {.CHI_VERSION } from "../constants";

const redirectionScript = `
```

We can change this later. This does not need to be a function. It can directly be a string and `CHI_VERSION` can be imported from constants file (no need to pass it from `default.vue`).